src/logic/__test__/calculate.test.js

```diff
@@ -0,0 +1,48 @@
+import calculate from '../calculate';
+
+describe('calculator operations', () => {
+  it('expect 1 + 2 to be 3', () => {
+    const obj = {
+      total: '1',
+      next: '2',
+      operation: '+',
+    };
+    const btn = '=';
+    const result = calculate(obj, btn);
+    expect(result.total).toBe('3');
+  });
+
+  it('expect 3 - 1 to be 2', () => {
+    const obj = {
+      total: '3',
+      next: '1',
+      operation: '-',
+    };
+    const btn = '=';
+    const result = calculate(obj, btn);
+    expect(result.total).toBe('2');
+  });
+
+  it('expect 3 x 10 to be 30', () => {
+    const obj = {
+      total: '3',
+      next: '10',
+      operation: 'x',
+    };
+    const btn = '=';
+    const result = calculate(obj, btn);
+
+    expect(result.total).toBe('30');
+  });
+
+  it('expect 6 / 2 to be 3', () => {
+    const obj = {
+      total: '6',
+      next: '2',
+      operation: '÷',
+    };
+    const btn = '=';
+    const result = calculate(obj, btn);
+    expect(result.total).toBe('3');
+  });
+});
```

Comment:
- [ ] Good job writing these unit tests. However I notice that you are importing the `calculate.js` to test these operations, but you should import the `operate.js` for these operations as it is responsible for the operations.
Once you do this you will notice that you have the following missing tests:

- A test for the modulo `%` operation
- A test that you cannot find the modulo divided by `0`
- A test for unknown operations (i.e you cant operate on non numerical data)
- A test for you cant divide a number by 0
-